modules/gitops-release-notes-1-11-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-11-0_{context}"]
+= Release notes for {gitops-title} 1.11.0
+
+{gitops-title} 1.11.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-0_{context}"]
+== Errata updates
+
+[id="gitops-1-11-0-security-update-advisory_{context}"]
+=== RHSA-2023:NNNN - {gitops-title} 1.11.0 security update advisory
+
+Issued: 2023-12-07
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:NNNN[RHSA-2023:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, to view the container images in this release, run the following command:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-11-0_{context}"]
+== Fixed issues
+
+* Before this update, the Argo CD Notifications Controller did not support custom certificates added to `argocd-tls-certs-cm`. As a result, notification services with custom certificates did not receive notifications due to the "x509: certificate signed by unknown authority" error. This update fixes the issue by correctly initializing the cert resolver function in the Argo CD Notifications Controller to load all certificates stored in `argocd-tls-certs-cm`. Now, notification services with custom certificates can successfully receive notifications. link:https://issues.redhat.com/browse/GITOPS-2809[GITOPS-2809]
```

Comment:
Suggestion: Rewrite for clarity. What is `argocd-tls-certs-cm`? A file or config map? Please mention that wherever it comes in content.

Reply:
It's a configmap.

Reply:
ACK.

Resolved.